### PR TITLE
inkscape: add Dutch translation

### PR DESCRIPTION
**Pull request by JenniX3**
- N/A The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by navarroaxel** [comment]
Hi @JenniX3, you can send all the translation in one PR, we only request 1 commit per file. Thanks!

File: pages.nl/common/inkscape.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+`inkscape {{bestandsnaam.svg}} -i {{id}} -o {{object.png}}`
+
+- Exporteer een SVG-document naar PDF, converteer alle teksten naar paden:
+
+`inkscape {{bestandsnaam.svg}} -o {{bestandsnaam.pdf}} --export-text-to-path`
+
+- Dupliceer het object met id="pad123", roteer het duplicaat 90 graden, sla het bestand op, en sluit Inkscape af:
+
+`inkscape {{bestandsnaam.svg}} --select=path123 --verb="EditDuplicate;ObjectRotate90;FileSave;FileQuit"`
```
Comment (navarroaxel) [inline pages.nl/common/inkscape.md:33]: The argument options should be between `{{}}`, the same fix applies to the English page:
```suggestion
`inkscape {{bestandsnaam.svg}} --select=path123 --verb="{{EditDuplicate;ObjectRotate90;FileSave;FileQuit}}"`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
@JenniX3, thanks for your submission. Only 1 comment.

File: pages.nl/common/inkscape.md (hunk 1 above)
Comment (JenniX3) [reply pages.nl/common/inkscape.md:33]: I haven't used the Inkscape CLI myself, but it seems to me that this is part of the command as described above. Not a parameter template, or option?

If this was describing generically how to use the `--verb` argument to trigger actions in Inkscape, I would agree. But this section is specifically describing how to duplicate an object (`EditDuplicate`), rotate the object (`ObjectRotate90`), save the file (`SaveFile`), and quit Inkscape (`FileQuit`).

Please let me know if my understanding is incorrect.

Comment (navarroaxel) [reply pages.nl/common/inkscape.md:33]: @JenniX3 No, just check the last example in [tokei](https://github.com/tldr-pages/tldr/blob/master/pages/common/tokei.md). That's what I mean.

Comment (JenniX3) [reply pages.nl/common/inkscape.md:33]: Ahh, understood. Sure, I'll commit the suggestion then.
I'll also submit a PR for the English as well later.

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Pull request merged**